css/nixos-site.css

```diff
@@ -2,6 +2,8 @@
 
 body {
     padding-top: 50px;
+    /* a bit more mobile friendly */
+    font-size: 16px;
```

Comment:
Since this is intended for mobile devices, let's put it into a media query:

```
/* Landscape phone to portrait tablet */
@media (max-width: 767px) { ... }
 
/* Landscape phones and down */
@media (max-width: 480px) { ... }
```

My main issue is with how everything was made with the default font size in mind, with absolute font size units, which will only bring the unchanged font sizes up.

Reply:
I understand your cautiousness, but I've checked the rendered homepage and there don't seem to be problems.

Note that 16px is the default in Bootstrap 4, so I think this goes into the right direction making text a bit easier to read.

Reply:
It's [more involved than 16px](https://github.com/twbs/bootstrap/blob/09c99fe6e79ad397fdef262017d7b2bc49377b15/scss/_variables.scss#L278); basically it relies on the browser's settings, and most defaults will end up being equivalent to 16px.

While I do agree generally with the "default 16px" being better, my previous point still stands that everything has been conceived with bootstrap's default font-size in mind, not only our additions, but bootstraps' defaults.

This includes some subtle, but imo breaking changes like the line-height set in absolute units to 20px, and some misc paddings and margins. The line-height, especially, makes the website feel much more cramped, and lines squeezed together so closely might hamper readability more.

Reply:
Could.you provide an example of such occourance?

Reply:
I don't know of specifics for padding and margins, almost nothing is set side-by-side in the site, so that part is not as much an issue.

Though, most of the text in the site inherits the absolute unit line-height. 

> ![image](https://user-images.githubusercontent.com/132835/57170752-1213d380-6ddd-11e9-9a7a-b07e3456b3ff.png)
> ![image](https://user-images.githubusercontent.com/132835/57170760-235ce000-6ddd-11e9-9e62-8f93aa5fb752.png)

> ![image](https://user-images.githubusercontent.com/132835/57170819-72a31080-6ddd-11e9-8ebe-bd655b5852f0.png)
> ![image](https://user-images.githubusercontent.com/132835/57170822-75056a80-6ddd-11e9-8ebc-16ddc90c8f1d.png)

The reduced line spacing hampers readability, it should at least be increased accordingly. To be equivalent, it should be set to 23px on body.

Here's how it would look with a line-height of 23px on the body.

> ![image](https://user-images.githubusercontent.com/132835/57171056-ea256f80-6dde-11e9-9a64-6d0ce35f0f69.png)